btap/src/preprocessing.py

```diff
@@ -60,17 +64,16 @@ def read_output(path_elec,path_gas):
     Used to read the building simulation I/O file
 
     Args:
-        tenant: default value is standard
-        bucket: nrcan-btap
-        path_elec: file path where data is to be read from in minio. In the case of electric value, this would be path to the electric output file
-        path_gas: This would be path to the gas output file. This is optional 
+        path_elec: file path where data is to be read from in minio. This is a mandatory parameter and in the case where only one simulation I/O file is provided,  the path to this file should be indicated here. 
+        path_gas: This would be path to the gas output file. This is optional, if there is no gas output file to the loaded, then a value of path_gas ='' should be used
     Returns:
        btap_df: Dataframe containing the clean building parameters file.
        floor_sq: the square foot of the building
     """
 
     # Load the data from blob storage.
     s3 = acm.establish_s3_connection(acm.settings.MINIO_URL, acm.settings.MINIO_ACCESS_KEY, acm.settings.MINIO_SECRET_KEY)
+    logger.info("%s read_output s3 connection %s", s3)
```

Comment:
Two placeholders specified, but only one argument.

```suggestion
    logger.info("read_output s3 connection %s", s3)
```